server/models/articles.js

```diff
@@ -55,14 +62,24 @@ const getArticleModel = (sequelize, DataTypes) => {
       onDelete: "CASCADE"
     });
 
-    Article.hasMany(db["Favourites"], {
+    Article.hasMany(db["Favorites"], {
       foreignKey: "articleId",
       target: "id",
       hooks: true
     });
   };
 
+  Article.prototype.increaseFavoriteCount = async function() {
+    this.favouriteCount++;
+    this.save();
+    await this.reload;
+  };
```

Comment:
This method is not needed, use Sequelize model instance increment function instead 
For the decrement, use Sequelize model instance decrement function also